homeTask1/app.js

```diff
@@ -0,0 +1,111 @@
+const UserForm = ({ addUser }) => {
+    return (
+        <form onSubmit={addUser} autoComplete="off" style={{ margin: "10px" }}>
+            <div style={{ margin: "10px" }}>
+                <label>First Name
+          <input type="text" id="firstName"></input>
```

Comment:
please use "controlled input" pattern
https://reactjs.org/docs/forms.html

You can consult with Ivan's solution